recipe/meta.yaml

```diff
@@ -28,10 +28,9 @@ requirements:
     - make
   host:
     - spdlog
-    - thrust >=1.9.0
-  run:
-    # - spdlog >=1.8.5  # (version constraint set via run_exports)
-    - thrust >=1.9.0
```

Comment:
Appears `thrust` was removed. Is this intended?

Reply:
Yes, since RMM exports it in `include/rapids/thrust`. Should probably split that off into a separate `rapids-thrust` package or something of the sorts in the future, but it's not a problem for now.